tools/autograd/derivatives.yaml

```diff
@@ -800,6 +800,18 @@
   self: pinv_backward(grad, result, self)
   result: pinv_jvp(self_p, result, self_t)
 
+- name: _linalg_svd_rank_restricted_helper(Tensor input, Tensor? atol=None, Tensor? rtol=None, bool full_matrices=True, bool compute_unique_rank=True) -> (Tensor U, Tensor S, Tensor Vh, Tensor rank, Tensor unique_rank)
+  input: _linalg_svd_rank_restricted_helper_backward(grads, input, U, S, Vh, rank, unique_rank)
+  atol: non_differentiable
+  rtol: non_differentiable
```

Comment:
They are differentiable, aren't they?

Reply:
Not with the current backward implementation, no.

Reply:
Well, isn't their derivative just zero?

Reply:
No, because their value impacts the output.

Reply:
Sure, but if you move a little bit the `atol` and a little bit the `rtol`  the output won't change, so their derivative should be zero, right?